addons/rating/models/mail_thread.py

```diff
@@ -13,7 +13,7 @@ def message_post(self, **kwargs):
         message = super(MailThread, self).message_post(**kwargs)
         if rating_value:
             ir_model = self.env['ir.model'].sudo().search([('model', '=', self._name)])
-            self.env['rating.rating'].create({
+            self.env['rating.rating'].sudo().create({
```

Comment:
`sudo` on create on Public method  ? :no_good_man:

Reply:
It is legit because the message_post of mail.message check already access.